hw/dv/data/common_sim_cfg.hjson

```diff
@@ -52,6 +52,9 @@
   // Top level simulation entities.
   sim_tops: ["-top {tb}"]
 
+  // Fileset selection for templated modules (current options are top or ip)
+  fsel_opts: "ip"
```

Comment:
The filelist generation step itself uses the generically named `sv_flist_gen_cmd` and `sv_flist_gen_opts`, the idea being, another flow (such as bazel) can more easily replace fusesoc by supplying its own set of values. As such, any additional variables that eventually get expanded into the generic vars above (such as `fsel_opts`) and are very much tool specific should be prefixed with the tool name to make it clear that that it is only relevant to that particular tool. Please rename this to `fusesoc_fileset`.

Reply:
Or perhaps `fusesoc_fileset_flag`.

Reply:
let's go with `fusesoc_fileset` the other one is a bit too verbose.

Reply:
Actually on second thoughts, we could make this more generic such as `design_level` and you can set the default value in `hw/data/common_proj_cfg.hjson`. THis is imported by all flows, so this will reduce the number of files needed to be touched.